frontend/src/static/js/main.js

```diff
@@ -254,15 +254,12 @@ window.app = new window.Vue({
       return 'http://reposense.org';
     },
 
-    getRepoSenseVersionLink() {
+    getRepoSenseUserGuideVersionLink() {
       const version = window.app.repoSenseVersion;
       if (!version) {
         return 'https://github.com/reposense/RepoSense';
```

Comment:
Should this link also be changed?

Reply:
If the version doesn't exist, the URL will not work properly. That's why I think we should keep this as it is.

Reply:
Ok, understood.